### PR TITLE
Gevent

### PR DESCRIPTION
**Pull request by Elfoniok**
On windows gevent is used with uvent implementation.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ pystun==0.1.0
 pytz
 six==1.10.0
 Twisted>=17.1.0
-cffi>=1.9.1
+cffi>=1.10.0
```
Comment (mfranciszkiewicz) [inline requirements.txt]: Please change it to `==`. `cffi==1.10.0` is an explicit dependency of crossbar.

Comment (paulperegud) [reply requirements.txt]: done

**Review by paulperegud — APPROVED** [review]

**Review by badb — COMMENTED** [review]
Can we exclude gevent-reactor and uevent from codecov?

File: tests/golem/test_defer.py [hunk 2]
```diff
@@ -0,0 +1,3022 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
```
Comment (paulperegud) [inline tests/golem/test_defer.py]: License is the same one we are using?

Comment (badb) [reply tests/golem/test_defer.py]: Twisted is using MIT: https://pypi.python.org/pypi/Twisted

File: requirements.txt [hunk 3]
```diff
@@ -47,3 +46,7 @@ qt5reactor
 GitPython>=2.1.0
 semantic_version
 pyOpenSSL==16.2.0
+gevent>=1.2.1
+devp2p==0.9.1
```
Comment (badb) [inline requirements.txt]: Why are we going back to devp2p in this branch?

Comment (Elfoniok) [reply requirements.txt]: Done

**Pull request closed without merging**